client/src/pages/SearchResultsUser.js

```diff
@@ -1,23 +1,53 @@
-import React from "react";
+import React, {useEffect, useState} from "react";
 import { Typography } from "@material-ui/core";
-import SearchList from "../Components/SearchResList";
```

Comment:
If you're going to change the name up here, then you have to change it to match where the dependency is being called on Line #48 

It's also okay if they don't match the name exactly.